example/ios/Runner/Info.plist

```diff
@@ -22,12 +24,6 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>NSCameraUsageDescription</key>
-	<string>你的操作需要访问相机权限</string>
-	<key>NSMicrophoneUsageDescription</key>
-	<string>你的操作需要访问录像机权限</string>
-	<key>NSPhotoLibraryUsageDescription</key>
-	<string>你的操作需要访问照片库权限</string>
```

Comment:
Is it really okay to remove this?

Reply:
I’m  sorry I forgot to put it on place after I did generation code. If do this kind of mistake again please let me know. But I will try not to.